### PR TITLE
LPS-148669 Remove petra-mail

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #1346

### CODE REVIEW

File: modules/apps/petra/petra-mail/src/main/java/com/liferay/petra/mail/MailEngine.java [hunk 1]
```diff
@@ -80,18 +78,7 @@
 public class MailEngine {
 
 	public static Session getSession() {
-		Session session = null;
-
-		try {
-			session = MailServiceUtil.getSession();
-		}
-		catch (SystemException systemException) {
```
Comment (julschong) [inline modules/apps/petra/petra-mail/src/main/java/com/liferay/petra/mail/MailEngine.java:88]: changed from last pull

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/mail/test/MailEngineTest.java [hunk 2]
```diff
@@ -154,7 +162,16 @@ public void setLastResetTime(long lastResetTime) {
 			_lastResetTime.set(lastResetTime);
 		}
 
-		private LimitedMailEngine(
+		private TestMailEngine() throws Exception {
```
Comment (julschong) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/mail/test/MailEngineTest.java:165]: change from last pull.
now both method uses TestMailEngine

**Comment by tinatian** [comment]
@julschong I will take the code this time, next time, please merge the changes of your "Auto SF" commit to where you modified the code, which means, for each commit, you should make the change, run SF, commit the changes, then move on to the next one. 
@catarineaguiar Please help me scan test results.

Thanks.
Tina.

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine, I found out that LibraryReferenceTest is not enough to check a test case. I'll resubmit this

**Pull request closed without merging**